### ISSUE TITLE
Initial/final layer print temperature doesn't match desired print temperature 

### ISSUE THREAD

**Issue by Reynolds-Beta** [Ultimaker/Cura#14019]
### Application Version

5.2.1

### Platform

Windows 11

### Printer

Ultimaker S5

### Reproduction steps

1. Add a new material. In this case I duplicated generic PLA to create a MatterHackers MHBuild PLA material which (from experience) operates better from a bit lower temp than the generic settings
2. Set Default Printing Temperature in the material to 185C - this is the lower range of MHBuild PLA recommended temps.
3. Set Default Build Plate Temperature in the material to 40C. Again, as recommended
4. Active the new material profile, choosing to discard any settings from previous settings
5. Choose the Default Fine (0.1) profile - you can actually choose almost any "profile" and get a bunch of different temperatures _other_ than what you set.

See the material printing temperatures highlighted.

### Actual results

Printing and build-plate temperatures do not reflect what was entered - or at least there isn't an explanation as to why they are different and how they are controlled. See picture below.

Some of these values are now below the range of recommended settings from the manufacturer (see Initial/Final Layer Temperature) - in this case 180 is the low point of the range.

![image](https://user-images.githubusercontent.com/44441515/206917794-f3bed75d-8ccf-4091-8e57-6810c92689b4.png)

### Expected results

Entered values are reflected in settings. If such is the case, then the material settings (when making the material profile) should have the additional fields to set the calculated fields.

**and**

Explanations on why they are different to expected values.

### Checklist of files to include

- [ ] Log file
- [ ] Project file

### Additional information & file uploads

Choosing other profiles such as Default Extra Fast change the values to even different than Default Fine, again without explanation or documentation. See below.

![image](https://user-images.githubusercontent.com/44441515/206918103-466e7cce-4a09-4677-bb08-39b7380fe85e.png)

**Comment by Reynolds-Beta** [comment]
Appreciate the response, and I understand the situation - it's complex - isn't it always?

In this particular case - I'm not using a custom "profile", just the canned ones for the Ultimaker S5 (Visual, Engineering, Draft, etc.). This is always my starting point, after which if I need to tweak a setting, I'll do so _then_ save it as a custom profile. 

Let's break it down by the bugs in just this case:
- Ignoring initial/final layer temp, the **Printing Temperature** setting is 195, not 185 that I set
- Build plate is set to 60, not the 40 that I set
- Irrespective of whether they are included in gcode or not, the initial/final layer temp are incorrect,

Basically, the settings are all over the place. There may indeed be a good reason for these kinds of deviations, but it is non-obvious to the user what is driving them.
